contrib/flake.nix

```diff
@@ -113,15 +109,10 @@
               # ASAN_OPTIONS=detect_leaks=1
               export ASAN_OPTIONS="log_path=./test.log:abort_on_error=1"
               export UBSAN_OPTIONS=print_stacktrace=1
-              mkdir -p build/runtime/parser
-              # nvim looks into CMAKE_INSTALL_DIR. Hack to avoid errors
-              # when running the functionaltests
-              mkdir -p outputs/out/share/nvim/syntax
-              touch outputs/out/share/nvim/syntax/syntax.vim
```

Comment:
not exactly sure about those, but tests seemed to run fine without them

Reply:
yeah it was a dirty hack to work around some bug. Seems like it's not needed anymore . `make functionaltest TEST_FILE=/home/teto/neovim/neovim/test/functional/treesitter/parser_spec.lua` passes on your branch